formula_renames.json

```diff
@@ -24,5 +24,6 @@
   "plt-racket": "racket",
   "polarssl": "mbedtls",
   "screenbrightness": "brightness",
-  "wires": "geckodriver"
+  "wires": "geckodriver",
+  "racket": "minimal-racket"
```

Comment:
@MikeMcQuaid, @jawshooah: Thank you for the feedback.

@jawshooah correctly pointed out that the file currently is not in order. So you want me to sort the whole thing? Do I sort in terms of the old name (the key) or the new one (the value)? Shouldn't I do it in a separate Pull Request?